tests/system/python/helpers/plugin_and_service.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+"""Install plugin as per type, plugin name, language & add/start south service"""
+
+import subprocess
+import http.client
+import json
+
+
+def install(_type, plugin, branch="develop", plugin_lang="python", use_pip_cache=True):
+    if plugin_lang == "python":
+        path = "$FOGLAMP_ROOT/tests/system/python/scripts/install_python_plugin {} {} {} {}".format(
+            branch, _type, plugin, use_pip_cache)
+    else:
+        path = "$FOGLAMP_ROOT/tests/system/python/scripts/install_c_plugin {} {} {}".format(
+            branch, _type, plugin)
+    try:
+        subprocess.run([path], shell=True, check=True)
+    except subprocess.CalledProcessError:
+        assert False, "{} plugin installation failed".format(plugin)
+
+    # Cleanup /tmp repos
+    if _type == "notificationDelivery":
+        _type = "notify"
+    if _type == "notificationRule":
+        _type = "rule"
+        subprocess.run(["rm -rf /tmp/foglamp-service-notification"], shell=True, check=True)
+    subprocess.run(["rm -rf /tmp/foglamp-{}-{}".format(_type, plugin)], shell=True, check=True)
+
+
+def reset():
```

Comment:
should this be reset_plugins? Not sure if this should be added along with remove_directories fixture?